estilos/banner.css

```diff
@@ -0,0 +1,199 @@
+/*General labels*/
+.cards--body {
+  padding: 10px;
+  margin: 0;
+}
+
+.cards__container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr auto auto;
+  grid-template-areas:
+    "everlane1"
+    "harrys"
+    "casper"
+    "everlane2";
+  gap: 10px;
+  height: auto;
+  margin: 0 auto;
+  overflow: auto;
+}
+
+.cards--all {
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  transition: transform 0.3s;
+}
+
+.cards--all:hover {
+  transform: scale(1.05);
+}
+
+.cards--rectangular {
+  aspect-ratio: 3 / 4;
+  min-height: 300px;
+}
+
+/*Everything related to this label "a or buttons"*/
+.style--button {
+  text-decoration: none;
+  color: white;
+  background-color: rgba(0, 0, 255, 1);
+  padding: 10px;
+  border-radius: 5px;
+  width: max-content;
+  font-size: 12px;
+}
+
+.style--button:hover {
+  background-color: gray;
+}
+
+.btn--center {
+  align-self: center;
+}
+
+/*CARD 1*/
+.cards__card-everlane {
+  grid-area: everlane1;
+  background-image: url("../img/everlane1.jpg");
+  color: white;
+}
+
+.cards__card-everlane h2 {
+  font-size: 11px;
+  text-align: center;
+}
+
+.cards__card-everlane a {
+  color: white;
+  margin-top: auto;
+}
+
+/*CARD 2*/
+.cards__card-harrys {
+  grid-area: harrys;
+  background-image: url("../img/nav.png");
+  background-position: 30% center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.cards__card-harrys__container {
+  color: white;
+  margin-left: 90px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.cards__card-harrys p {
+  width: 60%;
+  text-align: center;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.cards__card-harrys a {
+  margin-top: auto;
+}
+
+/*CARD 3*/
+.cards__card-casper {
+  grid-area: casper;
+  background-image: url("../img/casper.webp");
+  color: rgb(4, 12, 12);
+  justify-content: flex-end;
+  text-align: center;
+  font-size: 25px;
+}
+
+.cards__card-casper p {
+  width: 40%;
+  justify-content: center;
+  margin: 10px auto;
+  font-weight: bolder;
+}
+
+/*CARD 4*/
+.cards__card-everlane-2 {
+  grid-area: everlane2;
+  background-image: url("../img/everlane2.webp");
+  color: azure;
+  text-align: center;
+}
+
+.cards__card-everlane-2 p {
+  display: flex;
+  width: 70%;
+  font-size: 18px;
+  align-items: center;
+  height: 100%;
+  margin: auto;
+  margin-bottom: 100px;
+}
+
+.cards__card-everlane-2 a {
+  margin-bottom: 100px;
+}
+
+/*TABLET*/
+@media (min-width: 678px) {
+  .cards__container {
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    grid-template-areas:
+      "everlane1 harrys"
+      "casper    everlane2";
+    height: auto;
+  }
+}
+
+/*DESKTOP*/
+@media (min-width: 992px) {
+  /*ESTA PARTE ES SI NO QUEREMOS SCROLL DEL BODY, IGUAL SE AGREGA, PRIENSO QUE NO,
+  YA QUE SOLO QUEREMOS EVITAR EL SCROLL EN EL BANNER DE CARDS
+  html,
+  body {
+    overflow-x: hidden;
+  }*/
```

Comment:
Delete this and figure out why the scroll is happening on smaller resolutions and fix it.